examples/tests/sdk_tests/std/array.w

```diff
@@ -22,32 +22,106 @@ test "at()" {
 }
 
 //-----------------------------------------------------------------------------
-// push()
+// push() & pop()
 
 let a = MutArray<str>["hello"];
 assert(a.length == 1);
 a.push("world");
 assert(a.length == 2);
 assert(a.at(0) == "hello");
 assert(a.at(1) == "world");
+a.pop();
+assert(a.length == 1);
+assert(a.at(0) == "hello");
 
-test "push()" {
+test "pushAndPop()" {
   let a = MutArray<str>["hello"];
   assert(a.length == 1);
   a.push("world");
   assert(a.length == 2);
   assert(a.at(0) == "hello");
   assert(a.at(1) == "world");
+  a.pop();
+  assert(a.length == 1);
+  assert(a.at(0) == "hello");
 }
 
 //-----------------------------------------------------------------------------
-// TODO: https://github.com/winglang/wing/issues/2785
-
 // concat()
+
+let b = MutArray<str>["hello"];
+assert(b.length == 1);
+let c = MutArray<str>["wing"];
+assert(c.length == 1);
+let d = b.concat(c);
+assert(d.length == 2);
+assert(d.at(0) == "hello");
+assert(d.at(1) == "wing");
+
+test "concat()" {
+  let b = MutArray<str>["hello"];
+  assert(b.length == 1);
+  let c = MutArray<str>["wing"];
+  assert(c.length == 1);
+  let d = b.concat(c);
+  assert(d.length == 2);
+  assert(d.at(0) == "hello");
+  assert(d.at(1) == "wing");
+}
+
+//-----------------------------------------------------------------------------
 // contains()
-// copy()
-// copyMut()
+
+let e = MutArray<str>["hello", "wing"];
+let f = "wing";
+let contains = e.contains(f);
+assert(contains);
+
+test "contains()" {
+  let e = MutArray<str>["hello", "wing"];
+  let f = "wing";
+  let contains = e.contains(f);
+  assert(contains);
+}
+
+//-----------------------------------------------------------------------------
 // indexOf()
+
+let g = MutArray<str>["hello", "wing"];
+let h = "wing";
+let index = g.indexOf(h);
+assert(index == 1);
+
+test "indexOf()" {
+  let g = MutArray<str>["hello", "wing"];
+  let h = "wing";
+  let index = g.indexOf(h);
+  assert(index == 1);
+}
+
+//-----------------------------------------------------------------------------
 // join()
+
+let i = MutArray<str>["hello", "wing"];
```

Comment:
Delimiter is optional (comma by default) - should add a test for that case as well